### PR TITLE
Salt crafting solved. Closes #1792

### LINKED ISSUE #1792: Salt crafting
**Problem:**

Difficult to mine salt. (for salads & bread and so on)

**Proposal:**
([by Sakamoto in Discord](https://discord.com/channels/268093825975713793/1305280157858398302))
Proposal adjusted, follow things will be done:
- [x] bucket salt (bucket:bucket_salt) -> salt block (lottores:salt_block) (#1794)
- [ ] bottle or glass with water (don't remember which one is filled from cauldron) -> salt 5 (lottores:salt)

### PR DESCRIPTION
**Pull request by lord-sakamoto**
**Описание PR:**

Adds a `lord_vessels:glass_bottle_salt` item, a simple texture for it, and related recipes.

**Рекомендации к тесту:**

Tested, it works for me.

**Дополнительная информация:**

This should completely fix the issue #1792

### CODE REVIEW

File: mods/lord/Tools/lord_vessels/src/bottles.lua [hunk 1]
```diff
@@ -33,3 +33,43 @@ minetest.register_node('lord_vessels:glass_bottle_water', {
...
+	selection_box   = {
+		type  = 'fixed',
+		fixed = { -0.25, -0.5, -0.25, 0.25, 0.4, 0.25 }
+	},
+	groups          = { vessel = 1, dig_immediate = 3, attached_node = 1 },
+	sounds          = default.node_sound_glass_defaults(),
+})
+
+minetest.register_craft({
+	type = "cooking",
```
Comment (alek13) [inline mods/lord/Tools/lord_vessels/src/bottles.lua:55]: please use single quotes everywhere

**Review by alek13 — CHANGES_REQUESTED** [review]
lgtm

File: mods/lord/Tools/lord_vessels/src/bottles.lua [hunk 2]
```diff
@@ -33,3 +33,43 @@ minetest.register_node('lord_vessels:glass_bottle_water', {
 	groups          = { vessel = 1, dig_immediate = 3, attached_node = 1 },
 	sounds          = default.node_sound_glass_defaults(),
 })
+
+local glass_bottle_salt_texture = 'lord_vessels_salt_bottle.png'
+minetest.register_node('lord_vessels:glass_bottle_salt', {
+	description     = S('Glass Bottle (Salt)'),
```
Comment (alek13) [inline mods/lord/Tools/lord_vessels/src/bottles.lua:39]: Also you forgot to add the translations in `locale` folder

File: mods/lord/Tools/lord_vessels/src/bottles.lua (hunk 1 above)
Comment (lord-sakamoto) [reply mods/lord/Tools/lord_vessels/src/bottles.lua:55]: Should I change them and commit?

Comment (alek13) [reply mods/lord/Tools/lord_vessels/src/bottles.lua:55]: yes. Just new commit into the same branch

**Review by alek13 — APPROVED** [review]

**Pull request merged**